test/fixtures/clustered-server/app.js

```diff
@@ -21,8 +21,8 @@ if (cluster.isMaster) {
     if (msg.type === 'getpids') {
       const pids = [];
       pids.push(process.pid);
-      for (var key in cluster.workers)
-        pids.push(cluster.workers[key].process.pid);
+      for (const worker in cluster.workers.values())
```

Comment:
Should this change to `MapPrototypeValues` as above?

Reply:
For fixtures that is controlled by us. I don't think there is a need to change here.